stablehlo/tests/interpret_divide.mlir

```diff
@@ -0,0 +1,57 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
+
+// CHECK-LABEL: Evaluated results of function: div_op_test_ui64
+func.func @div_op_test_ui64() -> tensor<4xui64> {
```

Comment:
I believe there's no guideline for ordering tests other than the generic types (bool, integer, floating-point, and complex), so let's order the rest alphabetically: i.e., `si64`, `ui64`, `f64`, and `c128`.